wire-ios-data-model/Tests/Source/Model/Conversation/ZMConversationTests+SelfConversation.swift

```diff
@@ -21,7 +21,7 @@ import Foundation
 
 class ZMConversationTests_SelfConversation: ZMConversationTestsBase {
 
-    private func createMLSSelfConversation(in context: NSManagedObjectContext) -> ZMConversation {
+    private func createMLSSelfConversation() -> ZMConversation {
         let conversation = ZMConversation.insertNewObject(in: uiMOC)
```

Comment:
I would prefer to have a pure function where there is no direct usage of `uiMOC` but just the required values are passed in like `ZMConversation.insertNewObject(in: context)`.
It's easier to read and understand with the same layer of abstraction on the outer scope, for example:

```swift
// given
let conversation = createMLSSelfConversation(context: uiMOC)

// when
let messages = await uiMOC.perform { conversation.messages }

// then
...
```